service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -71,6 +73,7 @@
 @Controller
 public class RunSetsApiController implements RunSetsApi {
 
+  private final ControllerUtils controllerUtils;
```

Comment:
Adding ControllerUtils should be the only necessary update to any existing controllers - this also means we can modify ControllerUtils fields without worrying about updating every usage